measures/2018/measures-data.json

```diff
@@ -47199,6 +47199,492 @@
     "subcategoryId": "patientSafetyAndPracticeAssessment",
     "cehrtEligible": false
   },
+  {
```

Comment:
@sarahnw Can you verify the 2018 CAHPS measures (for non-ACO and ACO) here?  In general, when not specified in the JIRA ticket I maintained the field values from 2017.